app/resources/forms.js

```diff
@@ -367,3 +367,13 @@ function make_form_expandable(expandedSection) {
     return false;
   });
 }
+
+function form_confirmation() {
+   var personName = $('given_name').value + ' ' + $('family_name').value;
```

Comment:
We change the order of given/family name with config.family_name_first per repository. Use it to change the order like this:
https://github.com/google/personfinder/blob/master/app/resources/create.html.template#L55